uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1781,6 +1781,41 @@ sub spool  {
            'MRIProcessingUtility.pm', $upload_id, $error, $verb);
 }
 
+=pod
+
+=head3 isValidMRIProtocol()
+
+Ensures no column in the C<mri_protocol> table has comma-separated values.
+
+RETURNS: 1 on success, 0 on failure
+
+=cut
+
+sub isValidMRIProtocol  {
+    my $this = shift;
+
+    my $query = "SELECT COUNT(*) FROM mri_protocol 
+                WHERE TR_range LIKE '%,%'
+                OR TE_range LIKE '%,%'
+                OR TI_range LIKE '%,%'
+                OR slice_thickness_range LIKE '%,%'
+                OR xspace_range LIKE '%,%'
+                OR yspace_range LIKE '%,%'
+                OR zspace_range LIKE '%,%'
+                OR xstep_range LIKE '%,%'
+                OR ystep_range LIKE '%,%'
+                OR zstep_range LIKE '%,%'
+                OR time_range LIKE '%,%'";
```

Comment:
I should add `mri_protocol_checks` and `mri_violations_log` to this check as well.
I can do it now for the first table, but for the second, I need to wait on @llevitis to see her PR on the Loris side first.

Reply:
I did `mri_protocol_checks` in PR #318